### DISCUSSION TITLE
part-ii/2019-85/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#315] [Question comments]
Part II 2019 Linear Analysis: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2019-85/

**Comment by stevezshen** [comment]
Does anyone have a hint for (c) $T_5$?

**Reply by asheshbati** [reply]
Work with a sequence on the unit ball, then it should come out nicely as it's dominated by the geometric sequence - it's bounded for all p.
